.github/workflows/front-cd.yml

```
@@ -0,0 +1,32 @@
+  deploy:
+    runs-on: ubuntu-latest # 작업이 실행될 환경
+    steps:
+    - name: 체크아웃
+      uses: actions/checkout@v3
+    - name: 도커허브에 로그인
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USER_NAME }}
+        password: ${{ secrets.DOCKER_USER_PW }}
+    - name: 이미지 빌드
+        run: docker buildx build --platform linux/amd64 -t ${{ secrets.DOCKER_USER_NAME }}/${{ secrets.DOCKER_IMAGE_NAME }}-fe:latest .
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 19: mapping values are not allowed in this context

(syntax-check)